components/molecule/select/src/index.js

```diff
@@ -197,6 +197,9 @@ MoleculeSelect.propTypes = {
   /** if select accept single value or multiple values */
   multiselection: PropTypes.bool,
 
+  /** if multiselection, limit the number of selected values */
+  maxTags: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
```

Comment:
💡 What do you think about this? Should we allow numbers only❓

Reply:
what could be a string for, here?

Reply:
IMO it's not needed to allow the user to pass the prop as a string since it's only a number representation